src/mapReducers.js

```diff
@@ -1,4 +1,5 @@
 import _ from 'lodash';
+import { isTargetAllAction } from './context';
```

Comment:
I would expect to define exported const for key that is by convention defined as "apply action to all keys in state". 
Something like `all = "@@map-reducers/all"` and if keySelector returns such key, then mapReducers reducers must apply action to each key in state.